### ISSUE TITLE
Compilation error

### ISSUE THREAD

**Issue by wm4** [dxx-rebirth/dxx-rebirth#64]
Well, this is more a question...
```
similar/main/kconfig.cpp:81:12: error: no member named 'plus' in namespace 'std'
using std::plus;
      ~~~~~^
```
What's the magical sconf option that makes this go away?

**Comment by btb** [comment]
This will fix it: https://github.com/btb/dxx-rebirth/commit/a8f4f25130451a516cc831d3eca93bb797a1c715

**Comment by vLKp** [comment]
When reporting compilation failures, please post some information about your environment.  It looks like btb has seen this failure before and recognized it.  If he had not, you would be on your own, since the code works fine in my standard configurations.

SConf options are for tuning the results of configure tests.  SCons options change whether features are enabled.  There are currently no options designed to add/remove build breaks.

**Comment by wm4** [comment]
>SConf options are for tuning the results of configure tests. SCons options change whether features are enabled. There are currently no options designed to add/remove build breaks.

With using modern C++ features, compilation seems to break left and right out of the box on various systems. I had to manually remove ``-Werror``, and had to "disable" another compilation failure using an extremely obscure scons flag (that one was already reported). So I assumed this was similar and probably well-known.

**Comment by vLKp** [comment]
Fixed in 630206275df67678b0f65c8f05b1a58491dac7aa.

**Comment by vLKp** [comment]
> With using modern C++ features, compilation seems to break left and right out of the box on various systems. I had to manually remove `-Werror`, and had to "disable" another compilation failure using an extremely obscure scons flag (that one was already reported). So I assumed this was similar and probably well-known.

My experience has been that, aside from some obscure compiler bugs, compiler support is good for the changes I publish.  I intentionally do not set `-Werror` as part of the standard build since compilers change warnings over time (but it is in my `site-local.py`).  I fix build errors as I find them, including warnings promoted to errors by my local `-Werror`.  If you use a modern environment and are still getting errors, you should report them.  If you do not report them, I will not know to fix them, so you will keep hitting them every time you update.

If you are referring to [`sconf_cxx11_inherit_constructor=force-failure`](https://github.com/dxx-rebirth/dxx-rebirth/issues/62#issuecomment-96232289), then it is documented in the `scons --help` output.  However, as I noted when btb reported that it made the error go away, that was not supposed to happen.  `sconf_cxx11_inherit_constructor` is not supposed to change the outcome of a compile.  Rather, it is provided to override configure tests, so that I can check the results of a fallback implementation without keeping legacy compilers that only work with that fallback.
